applications/photo-asset-manager/README_TEMPLATE.md

```diff
@@ -0,0 +1,205 @@
+
+![AWS Photo Analyzer](images/pic3.png)
+
+The PAM application uses a lifecycle rule that places all images into an **Intelligent-Tiering** storage class at midnight. An Amazon S3 lifecycle configuration is a set of rules that define actions that Amazon S3 applies to a group of objects. For more information about an Amazon S3 lifecycle, see [Managing your storage lifecycle](https://docs.aws.amazon.com/AmazonS3/latest/userguide/object-lifecycle-mgmt.html).
+
+The following illustration shows Amazon S3 objects that are stored in the storage class.
+
+![AWS Photo Analyzer](images/storageMessage2.png)
+
+**Note**: For more information about Amazon S3 storage classes, see [Amazon S3 Storage Classes](https://aws.amazon.com/s3/storage-classes/).
+
+To retrieve images, select the given labels and choose **Download**.
+
+![AWS Photo Analyzer](images/pamDownload.png)
+
+The application dynamically creates a .zip file, stores the file in the Amazon S3 working bucket, presigns the file, and sends a notification message by using Amazon SNS. The message contains the presigned URL that's used to download the file.
+
+![AWS Photo Analyzer](images/email.png)
+
+### Understand the AWS resources used by the PAM application
+
+This section describes the AWS resources that the PAM application uses. You do not have to manually deploy any of these AWS resources, such as the AWS Lambda functions, by using the AWS Management Console. Instead, you can deploy all of them by running a provided AWS CDK script. Instructions on how to deploy these AWS resources are provided later in this document.
+
+#### AWS Lambda functions
+
+The backend of the PAM application is implemented by using these AWS Lambda functions:
+
+- **Upload** - Uploads an image to an S3 bucket by using a presigned URL.
+- **DetectLabelsFn** - An Amazon S3 trigger function that is invoked when an image is uploaded or copied to the S3 storage bucket.
+- **LabelsFn** - Returns the labels and label count of images in the S3 storage bucket.
+- **PrepareDownload** - Retrieves the keys of all images that correspond to the labels in the JSON data. Places all images in a zip file, presigns the zip file, and sends a notification message using Amazon SNS. The message contains a link to the zip file.
+
+**Note**: These AWS Lambda names are short names. The full names that appear in the AWS Management Console depend on how you configure the provided AWS CDK script. Full names appear as {PAM_NAME}{Function Name}. For example, **MyApp-LambdasDetectLabelsFn**.
+
+The following table describes the AWS Lambda functions used by this application.
+| Function |Trigger | Input | Output | Uses |
+| ------------- | ---------------------- | ---------------------------------| --------------------------------------------| ------------------------------------|
+| Upload | APIG PUT /upload | See following example | See following example | Storage bucket |  
```

Comment:
In some places you use "Labels table" and in others "Label table", could it be consistent?

Reply:
Done